src/pages/graphql/schema/product-recommendations/queries/recommendations.md

```diff
@@ -0,0 +1,240 @@
+                                "promotions/pants-all"
+                            ],
+                            "visibility": "Catalog, Search"
+                        },
+                        {
+                            "sku": "WP02",
+                            "rank": 4,
+                            "score": 44.674534,
+                            "queryType": "primary",
+                            "categories": [
+                                "women/bottoms-women",
+                                "women/bottoms-women/pants-women",
+                                "promotions/pants-all",
+                                "collections/performance-fabrics"
+                            ],
+                            "visibility": "Catalog, Search"
+                        },
+                        {
+                            "sku": "WP01",
+                            "rank": 5,
+                            "score": 41.731136,
+                            "queryType": "primary",
+                            "categories": [
+                                "women/bottoms-women",
+                                "women/bottoms-women/pants-women",
+                                "promotions/pants-all",
+                                "collections/performance-fabrics"
+                            ],
+                            "visibility": "Catalog, Search"
+                        }
+                    ]
+                }
+            ]
+        }
+    }
+}
+```
+
+## Input fields
+
+Field | Data type | Description
+--- | --- | ---
+`cartSKUs` | [String] | SKUs of the products in the cart.
+`category` | String | The category currently being viewed.
+`currentSKU` | String |  SKU of the product currently being viewed on the product detail page.
+`pageType` | PageType | Type of page on which recommendations are requested.
+`userPurchaseHistory` | [PurchaseHistory] | User purchase history with timestamp.
+`userViewHistory` | [ViewHistory] | User view history with timestamp.
+
+### PurchasedHistory
+
+Field | Data type | Description
+--- | --- | ---
+`date` | DateTime | Date of purchase
+`items` | [String]! | Items purchased
+
+### ViewHistory
+
+Field | Data type | Description
+--- | --- | ---
+`date` | DateTime | Date the items were viewed
+`sku` | [String]! | SKU of items viewed
+
+## Output fields
+
+The `Recommendations` object contains details about recommended products for a given SKU. It contains the following attributes.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`results` | [RecommendationUnit] | List of recommendation units for recommended products
+`totalResults` | Int | Total number of recommendation units for returned recommendation
+
+### RecommendationUnit attributes
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`displayOrder` | Int | Order in which recommendation units are displayed
+`pageType` | String | Page Type
+`productsView` | [ProductView] | List of product view
```

Comment:
There's a ProductView interface in Catalog Service, but it contains a different set of fields. Is the Recommendation schema completely separate from Catalog Service/Live Search?